secureboot_policy.go

```diff
@@ -655,13 +655,14 @@ func isShimExecutable(r io.ReaderAt) (bool, error) {
 	return pefile.Section(".vendor_cert") != nil, nil
 }
 
-// secureBootProtectionParams provide the arguments to computeSecureBootPolicyDigests.
-type secureBootProtectionParams struct {
-	loadSequences []*EFIImageLoadEvent // A list of EFI image load sequences for which to compute PCR digests for
+// EFISecureBootPolicyProfileParams provide the arguments to AddEFISecureBootPolicyProfile.
+type EFISecureBootPolicyProfileParams struct {
+	PCRAlgorithm  tpm2.HashAlgorithmId
+	LoadSequences []*EFIImageLoadEvent // A list of EFI image load sequences for which to compute PCR digests for
```

Comment:
now that this is exported, it would be better documented in the style of:

// LoadSequences is a list ...

before the def

Reply:
I've updated this now, and added some documentation for PCRAlgorithm too.